ui/src/components/select/QSelect.js

```diff
@@ -331,6 +331,12 @@ export default Vue.extend({
   },
 
   methods: {
+    getEmitValue() {
```

Comment:
Did u mean `getEmitValue (opt)`?

Reply:
Undoubtedly yes! It was a quick 'n dirty fork/edit/PR flow. Updating right now.